arviz/plots/backends/bokeh/autocorrplot.py

```diff
@@ -4,14 +4,22 @@
 from bokeh.layouts import gridplot
 from bokeh.models.annotations import Title
 
+from . import backend_kwarg_defaults
```

Comment:
Because this is run when autocorr is imported, and because in __init__ autocorr is above definition, python can't import. Like other comment move `backend_kwarg_defaults` above imports in the __init__ files and everything should be good